internal/utils/ptr/ptr.go

```diff
@@ -0,0 +1,10 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package ptr
+
+func To[T any](in T) *T {
```

Comment:
We should use this to clean up a bunch of the misc test files at some point that declare a ton of ints just to pass the pointer when declareing xDS in-line.

Reply:
sg, will raise a issue for this